web/src/beta/features/Dashboard/ContentsContainer/Projects/Project/hooks.ts

```diff
@@ -92,11 +97,23 @@ export default ({
     },
     [isStarred, onProjectUpdate, project]
   );
+ 
+  const filteredStories = useMemo(() => {
+    return stories?.map(
+      story => ({ publishmentStatus: toPublishmentStatus(story.publishmentStatus) })
+    );
+  }, [stories]);
```

Comment:
I think we don't need two useMemos for status, we can put this part into the useMemo where we get the storyStatus. 
(maybe rename `storyStatus` to `hasStoryPublished`, `publishStatus` to `hasMapPublished` as well)